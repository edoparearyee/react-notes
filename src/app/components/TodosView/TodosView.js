import React, { Component } from 'react';
import Typography from '@material-ui/core/Typography';
import Card from '@material-ui/core/Card';
import CardContent from '@material-ui/core/CardContent';

import './TodosView.scss';
import TodoAddFormContainer from '../../containers/TodoAddForm/TodoAddForm';
import TodoListContainer from '../../containers/TodoList/TodoList';

class TodosView extends Component {
  render() {
    return (
      <div className="TodosView">
        <Typography component="h1" variant="h4" gutterBottom>
          Todos
        </Typography>
        <Card>
          <CardContent>
            <TodoListContainer />
          </CardContent>
        </Card>
        <TodoAddFormContainer />
      </div>
    );
  }
}

export default TodosView;
