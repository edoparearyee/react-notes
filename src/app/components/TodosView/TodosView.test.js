import React from 'react';
import { shallow } from 'enzyme';

import TodosView from './TodosView';

describe('TodosView', () => {
  it('renders without crashing', () => {
    const component = shallow(<TodosView />);
    expect(component).toMatchSnapshot();
  });
});
