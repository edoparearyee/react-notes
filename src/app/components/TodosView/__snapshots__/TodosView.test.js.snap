// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodosView renders without crashing 1`] = `
<div
  className="TodosView"
>
  <WithStyles(Typography)
    component="h1"
    gutterBottom={true}
    variant="h4"
  >
    Todos
  </WithStyles(Typography)>
  <WithStyles(Card)>
    <WithStyles(CardContent)>
      <Connect(TodoList) />
    </WithStyles(CardContent)>
  </WithStyles(Card)>
  <Connect(TodoAddForm) />
</div>
`;
